README.md

```diff
@@ -484,35 +484,35 @@ HubProxy can be configured using either command-line flags or a YAML configurati
 
 Sensitive configuration values should be provided through environment variables:
 
-- `GITHUB_WEBHOOK_SECRET`: GitHub webhook secret for verification (required)
+- `HUBPROXY_WEBHOOK_SECRET`: GitHub webhook secret for verification (required)
 
 ### Configuration File
 
 Create a `config.yaml` file (see `config.example.yaml` for a template) with your desired settings.
 
 ```yaml
 # Target URL to forward webhooks to
-target_url: "http://your-service:8080/webhook"
+target-url: "http://your-service:8080/webhook"
```

Comment:
Not sure how to get viper to be more flexible with config vs flag names. Right now it's all the same.